src/app/static/src/tests/components/runWorkflow/runWorkflowProgress.test.ts

```diff
@@ -18,4 +90,53 @@ describe(`runWorkflowProgress`, () => {
         await wrapper.setProps({workflowMetadata: workflowMeta})
         expect(wrapper.vm.$props.workflowMetadata).toBe(workflowMeta)
     })
+
+    it(`it can render reports table`, async () => {
+        const wrapper = getWrapper()
+        
+        await Vue.nextTick()
+        await Vue.nextTick()
+        await Vue.nextTick()
+        await wrapper.setData({workflowRunStatus: workflowStatus1})
```

Comment:
I was unable to interact with v-select directly with unit tests so have covered the format of the items in the dropdown and the behaviour when an item is selected in the selenium tests instead.

Reply:
I had tried using a full mount but I couldn't get that to work either unfortunately.